app/views/msg/info.erb

```diff
@@ -1,13 +1,19 @@
-Event Name: <%=event.title%> (<%= event.id_code %>)
-Duration: <%= event.start_time %> to <%= event.end_time %>
+Name: <%=event.title%> (<%= event.id_code %>)
+<%= event.start_time %> to <%= event.end_time %>
 Description: <%= event.description %>
 <%
   avp  = event.available_people
   pavp = event.partially_available_people
   uavp = event.unavailable_people
   lame = event.unresponsive_people
+  short_date_time = lambda { |from, to| "#{from.strftime('%m/%d %H%M')}-#{to.strftime('%H%M')}" }
 %>
 Available: <%= avp.count %>
 Partially Available: <%= pavp.count %>
 Unavailable: <%= uavp.count %>
 Unknown: <%= lame.count %>
+<% if event.people.include?(person) %>
+Your Assignments: <% event.assignments.each do |asmt| %>
```

Comment:
What if we changed this to
`event.assignments.where(person: person).each` and then we wouldn't need the `if` later. That seems better than iterating over assignments we don't want.

Reply:
it took a while to get around model associations .. little confusing with the delegations

and yes that sounds good .. i'll push a commit with the update